keyboards/montsinger/rebound/rebound.h

```diff
@@ -26,7 +32,7 @@
  * represents the switch matrix.
  */
  
-#define LAYOUT_ortho_4x12( \
+#define LAYOUT_48( \
```

Comment:
Is this compatible with both rev1 and rev2?

Also, this should still be ortho_4x12 based
```suggestion
#define LAYOUT_ortho_4x12( \
```